api/service/syncing/syncing.go

```diff
@@ -637,6 +637,8 @@ func (ss *StateSync) getMaxPeerHeight() uint64 {
 		wg.Add(1)
 		go func() {
 			defer wg.Done()
+			//debug
+			//utils.GetLogInstance().Warn("[Sync] getMaxPeerHeight", "IP", peerConfig.ip, "Port", peerConfig.port)
```

Comment:
please remove the debug code or use Debug()

Reply:
I might use it later for debug